packages/zostso/__tests__/__system__/__scripts__/start/start_app_new_as.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+account=$1
+host=$2
+port=$3
+user=$4
+password=$5
+ru=$6
+zowe zos-tso start app --app-key "test2" --startup "EXEC 'CUST009.PUBLIC.REXX(TESTADRS)'" --account $account --host $host --port $port --user $user --password $password --ru $ru
```

Comment:
We need to create a data set, we cannot assume that the data set is going to be present on the system (CUST009.PUBLIC.REXX(TESTADRS)).